tools/check.py

```diff
@@ -200,7 +207,99 @@ def _validate_section_heading_order(self, ast_node=None, headings=None):
                 "the order specified by the template".format(self.filename))
 
         return (len(missing_headings) == 0) and \
-               valid_order and no_extra and correct_level
+            valid_order and no_extra and correct_level
+
+    def _validate_one_callout(self, callout_node):
+        """
+        Logic to validate a single callout (blockquoted section). Checks that:
+        * First child of callout box should be a header with known & CSS style
+        * Callout box must have at least one child after the heading
+        * Checks # times callout appears in document, min <= n <= max
```

Comment:
"min <= n <= max" -> "minc <= n <= maxc"

This will make easy to read the code.